src/core/Akka/Serialization/Serialization.cs

```diff
@@ -119,6 +119,26 @@ public Serialization(ExtendedActorSystem system)
             }
         }
 
+        private Serializer GetSerializerByName(string name)
+        {
```

Comment:
So this is the helper method for translating the serializer name to an actual instance. 
Downside, having to translate it here. Upside, being able to leverage caching once this is done.